hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegionFileSystem.java

```diff
@@ -850,6 +877,7 @@ public void commitMergedRegion(final RegionInfo mergedRegionInfo) throws IOExcep
         throw new IOException("Unable to rename " + mergedRegionTmpDir + " to "
             + regionDir);
       }
+      loadRegionFilesIntoStoreTracker(regionDir);
```

Comment:
I think we should also get the file list from store file tracker instead of listing?

Reply:
These would not be in the tracker yet, as these are all actually links to the regions original files.

Reply:
I mean, when splitting/merging, we will create the linked store files in memory, we should just use them to insert to the tracker, so we do not need to list again.